mod_lookup_identity.c

```diff
@@ -120,18 +157,18 @@ static int lookup_user_by_certificate(request_rec * r) {
 	reply = dbus_connection_send_with_reply_and_block(connection,
 		message, timeout, &error);
 	if (! reply || dbus_error_is_set(&error)) {
-		stage = "dbus_connection_send_with_reply_and_block(" DBUS_SSSD_IFACE_USERS "." DBUS_SSSD_FIND_BY_CERTIFICATE ")";
+		stage = "dbus_connection_send_with_reply_and_block(" DBUS_SSSD_IFACE_USERS "." DBUS_SSSD_FIND_BY_NAME_AND_CERTIFICATE ")";
 		goto fail;
 	}
 	int reply_type = dbus_message_get_type(reply);
 	if (reply_type != DBUS_MESSAGE_TYPE_METHOD_RETURN) {
-		stage = apr_psprintf(r->pool, DBUS_SSSD_IFACE_USERS "." DBUS_SSSD_FIND_BY_CERTIFICATE " returned [%d], not DBUS_MESSAGE_TYPE_METHOD_RETURN", reply_type);
+		stage = apr_psprintf(r->pool, DBUS_SSSD_IFACE_USERS "." DBUS_SSSD_FIND_BY_NAME_AND_CERTIFICATE " returned [%d], not DBUS_MESSAGE_TYPE_METHOD_RETURN", reply_type);
 		goto fail;
 	}
 
 	const char *path;
 	if (! dbus_message_get_args(reply, NULL, DBUS_TYPE_OBJECT_PATH, &path, DBUS_TYPE_INVALID)) {
```

Comment:
FindUserByNameAndCertificate returns single user or an error. So it will behave the same way as before.